Audio/FirstSound/main.go

```diff
@@ -26,12 +26,19 @@ func main() {
 }
 
 func generate() {
+	var (
+		start float64 = 1.0
+		end   float64 = 1.0e-4
+	)
 	nsamps := Duration * SampleRate
-	var angle float64 = tau / nsamps
+	var angle float64 = tau / float64(nsamps)
```

Comment:
Hello Dylan, Thanks for the code. Shouldn't this line be 
var angle float64 = tau/**SampleRate** ? , so that angle is independent of number of samples & only dependent on the sample rate.
BTW, i am not an expert, just trying to learn from https://dylanmeeus.github.io/posts/audio-from-scratch-pt2/